### ISSUE TITLE
Warning or error on duplicate definitions

### ISSUE THREAD

**Issue by annevk** [speced/bikeshed#2483]
I wonder how https://github.com/whatwg/url/issues/759 happened without the build script failing when the duplicate definition for "surrogate" got introduced. Is there a way for Bikeshed to pick a competing definition without complaining about it?

**Comment by tabatkins** [comment]
No, definitely not. Any clash between definitions is guaranteed to result in a Link Error complaining about it. I *did* quickly put in a predefined link-default preferring Infra over *all* i18n-glossary terms when there's a clash, but that would just mean that it silently chooses the Infra term, since it discarded the i18n-glossary one. (And I just confirmed, with a Bikeshed install that has older data that contains the i18n-glossary version of the term, that it does indeed silently link to Infra as expected.)

So I'm very confused about how URL ever picked up that anchor, unless it got built in the small interim period between when I switched to WebRef and when I inserted the Infra link-default, and somehow didn't die on the link error warning. Unfortunately there's no version control on the published versions so I'm not sure how to do archeology on that.
